R/html-tweak.R

```diff
@@ -324,6 +324,90 @@ badges_extract_text <- function(x) {
   xml <- xml2::read_html(x)
   badges_extract(xml)
 }
+
+activate_navbar <- function(html, path, pkg) {
+
+  html_navbar <- xml2::xml_find_first(html, ".//div[contains(@class, 'navbar')]")
+  nav_items <- xml2::xml_find_all(html_navbar,".//li[contains(@class, 'nav-item')]")
+
+  # external links can't be an active item
+  keep_internal <- function(links, pkg) {
+    links[!grepl("https?\\:\\/\\/", links) | grepl(pkg$meta$url %||% "", links)]
```

Comment:
`:` and `/` aren't special regexp characters?

What's the second condition testing?